### PR TITLE
Adds Doomsday Calendar

### PR DESCRIPTION
**Pull request by vampirebat74**
Adds an Abnormality from Canto I of Limbus Company

## About The Pull Request

Adds Doomsday Calendar

-Deals additional unavoidable RED damage when performing instinct work without a full counter.
- Instinct work raises the Qliphoth Counter.
- Qliphoth Counter of 5, lowers every time three abnormalities have been worked on (subject to change).
- Qliphoth counter can be raised without performing work by offering it fresh blood.
- Immobile when breaching, starts with extremely high resistances that lower as it is offered sacrifices.
- Failure to suppress it will lead to it unleashing massive waves of BLACK damage every 30 seconds, each of which reduce its HP by a bit under 25%

## Why It's Good For The Game

- Gives clerks something to do when they're not destroying the station with hydroponics.
- New and interesting way to make the most hated abnormalities.

## Changelog
:cl:
add: Added new things
soundadd: added a new sound thingy
imageadd: added some icons and images
/:cl:

### CODE REVIEW

File: code/game/objects/items/ego_weapons/he.dm
Comment (Coxswain-Navigator) [inline code/game/objects/items/ego_weapons/he.dm:761]: Might want to change this to fortitude or prudence, in accordance to the new temperance rebalance. Or maybe not. Talk to Kirie about it.

File: code/modules/clothing/suits/ego_gear/he.dm
Comment (Coxswain-Navigator) [inline code/modules/clothing/suits/ego_gear/he.dm:285]: Why not 0 white and 0 pale? Egor is going to have a fit.

Comment (Kitsunemitsu) [reply code/modules/clothing/suits/ego_gear/he.dm:285]: This is fine, 20 pale is on He, it's the average.

**Comment by Coxswain-Navigator** [comment]
Needs a test merge

File: code/modules/mob/living/simple_animal/abnormality/he/doomsday_calendar.dm
Comment (Kitsunemitsu) [inline code/modules/mob/living/simple_animal/abnormality/he/doomsday_calendar.dm:256]: if (!is_fed)
_but tbh I don't really think that the compiler cares_

**Review by Kitsunemitsu — APPROVED** [review]
One minor change; Will merge right after.

**Review by Kitsunemitsu — APPROVED** [review]
Approved; code looks fine and it's been testmerged for a while.

**Pull request merged**
